lib/screens/topic_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:maui/repos/article_repo.dart';
+import 'package:tuple/tuple.dart';
+
+class TopicScreen extends StatefulWidget {
+
+  TopicScreen({key}):super(key:key);
+  
+  @override
+  _TopicScreenState createState() => _TopicScreenState();
+}
+
+class _TopicScreenState extends State<TopicScreen> {
+
+  List<Tuple7<String, String, String, String, String, String, int >> _articles;
+
+  void _initTopic() async {
+    new ArticleRepo().getArticle('lion').then((_articles) async{
```

Comment:
How is _articles getting initialized here?

Reply:
The thought process while using _articles is as follow: This is have created to store a list of contents for particular topic and then treat as an array of contents. The element of list we will pass to the article class which rajesh is creating, he will take this data and render it on the screen . Now because I have dependency on Shantu for subcategories screen so i just created the screen and using "lion" id as harcoded as of now so that the dependency will not hinder or block the task assigned to me. As i will get the access to the topic id passed to me from subcategories page and once the data will be added into the database, i will use this _articles to initialise and store the data sir.
Thanks!